CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+# Official devRant Issue Tracker Rules
+
+Please follow these rules when you are:
+- Submitting a bug report
+- Asking for new features
+- Asking Questions
+
+
+## Submitting a bug report
+- Use the corresponding issue template
+- Please provide clear steps to reproduce the issue, if the steps are known
+- Make sure to include what platform(s) and version of the app you are able to reproduce the bug on
+
+
+## Asking for new features
+- Use the corresponding issue template
+- Please add "[Feature Request]" or "[Feature]" to the title
```

Comment:
I'm not sure if we need this? I'm thinking maybe we can just rely on tags for this. I think having the type in all of the titles might getting annoying/make them too long.

Reply:
Alright I just thought I could add this because there are a lot of those already.